BedrockNode.cpp

```diff
@@ -67,7 +63,6 @@ BedrockNode::BedrockNode(const SData& args, BedrockServer* server_)
     SINFO("BedrockNode constructor");
 }
 
-// --------------------------------------------------------------------------
```

Comment:
Why remove these?  I personally find it helpful to break up the sections visually, and would recommend we do it reliably rather than remove it.